contrib/parseq-retry/src/main/java/com/linkedin/parseq/retry/RetryFailureException.java

```diff
@@ -0,0 +1,17 @@
+package com.linkedin.parseq.retry;
+
+/**
+ * Exception that is thrown when the process of retries is interrupted by any reason.
+ * Usually it's either the exhausted number of attempts, or the unacceptable task result.
+ */
+public class RetryFailureException extends Exception {
+  private static final long serialVersionUID = 0L;
+
+  public RetryFailureException() {
+    super();
+  }
+
+  public RetryFailureException(final String message) {
+    super(message);
+  }
```

Comment:
Could you add a new constructor which takes a Throwable and calls super(Throwable t) so that we don't lose the stack trace when there is a fatal error?